utils/mil_ros_tools/online_bagger/nodes/online_bagger.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+
+import rospy
+import os
+import rosbag
+import rostopic
+from collections import deque
+import yaml
+import rospkg
+from std_srvs.srv import SetBool
+
+'''
+To call this service call the '/online_bagger' service with the SetBool Parameter set to True.
+Doing this will generate a rosbag with the last x number of seconds saved
+
+for example:
+
+    bagging_server = rospy.ServiceProxy('/online_bagger', SetBool)
```

Comment:
Filename would be a good idea. It could still just be one service but calling it with blank time field would make it dump everything